src/Cake.Core/IO/ProcessRunner.cs

```diff
@@ -52,6 +55,20 @@ public IProcess Start(FilePath filePath, ProcessSettings settings)
             // Get the arguments.
             var arguments = settings.Arguments ?? new ProcessArgumentBuilder();
 
+#if NETCORE
+            if (_environment.Platform.IsUnix() &&
```

Comment:
Also check running .NET Core here, as .NET standard in theory could be run under full framework too.